test/tstDBSCAN.cpp

```diff
@@ -177,28 +178,42 @@ void dbscan_f(ArborX::DBSCAN::Implementation impl)
   ArborX::DBSCAN::Parameters params;
   params.setImplementation(impl);
 
+  Kokkos::View<int *, MemorySpace> labels("Testing::labels", 0);
+
+#define VERIFY_DBSCAN(data, eps, minpts)                                       \
+  dbscan(space, (data), (eps), (minpts), labels, params);                      \
+  BOOST_TEST(verifyDBSCAN(space, (data), (eps), (minpts), labels));
```

Comment:
I think it might be better if you restricted the scope of labels
```C++
#define VERIFY_DBSCAN(data, eps, minpts) \
do { \
  Kokkos::View<int *, MemorySpace> labels("Testing::labels", 0); \
  dbscan(space, (data), (eps), (minpts), labels, params); \
  BOOST_TEST(verifyDBSCAN(space, (data), (eps), (minpts), labels)); \
} while (false)
```
(purposefully omitted the semicolon)